event-log/src/main/scala/io/renku/eventlog/init/DbInitializer.scala

```diff
@@ -46,6 +47,7 @@ class DbInitializerImpl[Interpretation[_]](
   override def run(): Interpretation[Unit] = {
     for {
       _ <- eventLogTableCreator.run()
+      _ <- eventPayloadTableCreator.run()
```

Comment:
Wouldn't it be better to add as the very last step. That would reflect the history of event-log DB evolution and I think should make things simpler